pkgs/by-name/pw/pwndbg/package.nix

```diff
@@ -7,47 +7,43 @@
 }:
 
 let
-  pythonPath = with python3.pkgs; makePythonPath [
-    capstone
-    future
-    psutil
-    pwntools
-    pycparser
-    pyelftools
-    pygments
-    unicorn
-    rpyc
-  ];
+  pythonPath = with python3.pkgs; makePythonPath [ pwndbg ];
+
   binPath = lib.makeBinPath ([
-    python3.pkgs.pwntools   # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/wrappers/checksec.py#L8
+    python3.pkgs.pwntools # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/wrappers/checksec.py#L8
   ] ++ lib.optionals stdenv.isLinux [
-    python3.pkgs.ropper     # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/ropper.py#L30
-    python3.pkgs.ropgadget  # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/rop.py#L32
+    python3.pkgs.ropper # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/ropper.py#L30
+    python3.pkgs.ropgadget # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/rop.py#L32
   ]);
-
-in stdenv.mkDerivation rec {
+in
+stdenv.mkDerivation rec {
   pname = "pwndbg";
-  version = "2022.12.19";
+  version = "2024.02.14";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "pwndbg";
     repo = "pwndbg";
     rev = version;
-    sha256 = "sha256-pyY2bMasd6GaJZZjLF48SvkKUBw3XfVa0g3Q0LiEi4k=";
-    fetchSubmodules = true;
+    sha256 = "sha256-/pDo2J5EtpWWCurD7H34AlTlQi7WziIRRxHxGm3K2yk=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
 
   installPhase = ''
+    runHook preInstall
+
     mkdir -p $out/share/pwndbg
-    cp -r *.py pwndbg gdb-pt-dump $out/share/pwndbg
+    cp gdbinit.py $out/share/pwndbg
     chmod +x $out/share/pwndbg/gdbinit.py
+
     makeWrapper ${gdb}/bin/gdb $out/bin/pwndbg \
       --add-flags "-q -x $out/share/pwndbg/gdbinit.py" \
       --prefix PATH : ${binPath} \
-      --set NIX_PYTHONPATH ${pythonPath}
+      --set PYTHONPATH ${pythonPath} \
+      --set PWNDBG_VENV_PATH PWNDBG_PLEASE_SKIP_VENV
```

Comment:
In next pwndbg release env will be removed (this is internal undocummented env) PWNDBG_VENV_PATH=PWNDBG_PLEASE_SKIP_VENV
Please create empty file .skip-venv